.ci/docker/dind-buildx/build-example.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# This script show how to use the dind-buildx Docker image to build a Docker image for multiple architectures.
+
+
+DOCKER_USER=${DOCKER_USER:?-"Missing environment variable."}
+DOCKER_PASSWORD=${DOCKER_PASSWORD:?-"Missing environment variable."}
+DOCKER_REGISTRY=${DOCKER_REGISTRY:-"docker.elastic.co"}
+
+docker run -it -e DOCKER_USER -e DOCKER_PASSWORD -e DOCKER_REGISTRY -v /var/run/docker.sock:/var/run/docker.sock -v "$(pwd):/app -w /app docker.elastic.co/observability-ci/dind-buildx:20.10.14" """
```

Comment:
I guess the tag is latest? https://github.com/elastic/apm-pipeline-library/pull/1689/files#diff-fb7f603de17a1d0f10946dc8155c9cd221f56fed83242cce81c8d81fbf0bae45R252

```suggestion
docker run -it -e DOCKER_USER -e DOCKER_PASSWORD -e DOCKER_REGISTRY -v /var/run/docker.sock:/var/run/docker.sock -v "$(pwd):/app -w /app docker.elastic.co/observability-ci/dind-buildx:latest" """
```

Reply:
both ara valid, I have pushed `20.10.14`for my tests, but the one generated every day will be `latest`